packages/lisk-transactions/src/transactions/helpers/get_bytes.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import * as cryptography from '@liskhq/lisk-cryptography';
+import BigNum from 'browserify-bignum';
+import { BYTESIZES } from '../../constants';
+import { TransactionJSON } from '../../transaction_types';
+
+export const getBytes = (
```

Comment:
im not sure if this function make sense.
Asset bytes always need to be included in the base bytes.
only in case of Transfer transaction, it's could be empty